RELEASE.md

```diff
@@ -16,15 +16,16 @@ Each bpfman project release is comprised of the following major components:
 - bpf-metrics-exporter and bpf-log-exporter binary crates
 - bpfman RPMs stored in the [bpfman COPR repository][copr-repo].
 - Kubernetes User Facing Custom Resource Definitions (CRDs)
-    - `BpfApplication`
+    - `BpfApplication` and `BpfNsApplication`
+    - `BpfProgram` and `BpfNsProgram`
```

Comment:
I think BpfProgram wasn't listed originally because users don't create them.  I suggest we leave it on the list and delete "User Facing" from line 18.

Reply:
Done